### PR TITLE
Version bump to pyyaml for CVE-2017-18342

### PR DESCRIPTION
**Pull request by Psykar**

### CODE REVIEW

**Comment by dtkav** [comment]
lol I made this exact PR.
I don't get why the lockfile doesn't contain the right version though...

**Comment by Psykar** [comment]
The lock file in the PR?
Or the lockfile before the PR?

pyyaml 5.1 was only released a couple of days ago. https://github.com/yaml/pyyaml/releases/tag/5.1

**Comment by dtkav** [comment]
look at the diff, if doesn't include pyyaml.

yeah I was working on updating connexion, openapi-spec-validar and this
last night.

I didn't publish the mc diff because I couldn't get it to actually update
the lock file.

**Comment by Psykar** [comment]
The diff does include pyyaml... 
https://github.com/fleetspace/missioncontrol/pull/7/files#diff-46bb8405eeb4d8a11deb635f562f7c48R561

File: Pipfile.lock [hunk 1]
```diff
@@ -547,19 +546,19 @@
...
+                "sha256:7a5d3f26b89d688db27822343dfa25c599627bc92093e788956372285c6298ad",
+                "sha256:9372b04a02080752d9e6f990179a4ab840227c6e2ce15b95e1278456664cf2ba",
+                "sha256:a5dcbebee834eaddf3fa7366316b880ff4062e4bcc9787b78c7fbb4a26ff2dd1",
+                "sha256:aee5bab92a176e7cd034e57f46e9df9a9862a71f8f37cad167c6fc74c65f5b4e",
+                "sha256:c51f642898c0bacd335fc119da60baae0824f2cde95b0330b56c0553439f0673",
+                "sha256:c68ea4d3ba1705da1e0d85da6684ac657912679a649e8868bd850d2c299cce13",
+                "sha256:e23d0cc5299223dcc37885dae624f382297717e459ea24053709675a976a3e19"
             ],
-            "version": "==3.13"
+            "version": "==5.1"
```
Comment (Psykar) [inline Pipfile.lock:561]: Note the change here.

Comment (Psykar) [reply Pipfile.lock:561]: (@dtkav )

**Comment by dtkav** [comment]
ah yeah there it is... oops

**Review by dtkav — APPROVED** [review]

**Pull request merged**